Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+import SwiftUI
+
+public typealias SliderValue = BinaryFloatingPoint & Strideable & CustomStringConvertible
+
+/// Enables selection from a range of values.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/slider/)
+public struct Slider<Value>: View where Value: SliderValue, Value.Stride: BinaryFloatingPoint {
+
+    enum SelectedValue {
+        case single(Binding<Value>?)
+        case range(Binding<ClosedRange<Value>>?)
+    }
+
+    @Environment(\.sizeCategory) var sizeCategory
+
+    let value: SelectedValue
+    let valueLabel: String
+    let startLabel: String
+    let endLabel: String
+    let bounds: ClosedRange<Value>
+
+    @State private var isFirstThumbDrag = false
+    @State private var isSecondThumbDrag = false
+    @State private var firstValue: Value {
+        didSet {
+            switch value {
+                case .single(let currentValue):
+                    currentValue?.wrappedValue = firstValue
+                case .range:
+                    updateSelectedRangeIfNeeded()
+            }
+        }
+    }
+
+    @State private var secondValue: Value? {
+        didSet {
+            updateSelectedRangeIfNeeded()
+        }
+    }
+
+    @State private var dragStartOffset = 0.0
+
+    private let buckets: [Range<Value>]
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
```

Comment:
The leading side seems to have wrong rounding of selected area

<img width="97" alt="image" src="https://user-images.githubusercontent.com/35844477/235096161-222f41c8-8819-4746-b493-57ceb942251f.png">

Reply:
Nice catch, missed that. Applied cornerRadius on progress as well.